fbpic/lpa_utils/laser/longitudinal_laser_profiles.py

```diff
@@ -229,40 +233,71 @@ def __init__(self, z0, spectrum_file, propagation_direction=1):
             in the file. The spectral phase can be omitted, in which case it
             will be assumed to be 0.
 
+        phi2_chirp: float (in second^2), optional
+            The amount of temporal chirp, at focus (in the lab frame).
```

Comment:
Could you add a word on the sign convention here? (i.e. positive `phi2_chirp` leads to blue/red in the front of the pulse)

Reply:
```suggestion
            The amount of additional temporal chirp, at focus (in the lab frame).
```
Could you also add a sentence explaining that this is *added* to the phase provided in the file?